app/src/main/java/com/framgia/englishconversation/screen/createPost/CreatePostViewModel.java

```diff
@@ -483,78 +473,67 @@ private void selectImage() {
     }
 
     public void onAudioConversationClick(final ConversationModel conversations) {
         showAudioDialog();
         RecordingAudioDialog.OnRecordingAudioListener recordingAudioClickListener =
-            new RecordingAudioDialog.OnRecordingAudioListener() {
-                @Override
-                public void onRecordingAudioClick(String filePath, String fileName) {
-                    MediaModel record = new MediaModel(MediaModel.MediaType.AUDIO);
-                    record.setId(UUID.randomUUID().toString());
-                    record.setUrl(filePath);
-                    record.setName(fileName);
-                    initMedia(filePath);
-                    conversations.setMediaModel(record);
-                }
+                new RecordingAudioDialog.OnRecordingAudioListener() {
+                    @Override
+                    public void onRecordingAudioClick(String filePath, String fileName) {
+                        MediaModel record = new MediaModel(MediaModel.MediaType.AUDIO);
+                        record.setId(UUID.randomUUID().toString());
+                        record.setUrl(filePath);
+                        record.setName(fileName);
+                        conversations.setMediaModel(record);
+                    }
 
-                @Override
-                public void onRecordCancel() {
-                    // no ops
-                }
-            };
+                    @Override
+                    public void onRecordCancel() {
+                        // no ops
+                    }
+                };
         mRecordingAudioDialog.setOnRecordingAudioClickListener(recordingAudioClickListener);
     }
 
     private void onAudioRecordingClick() {
         showAudioDialog();
         RecordingAudioDialog.OnRecordingAudioListener recordingAudioClickListener =
-            new RecordingAudioDialog.OnRecordingAudioListener() {
-                @Override
-                public void onRecordingAudioClick(String filePath, String fileName) {
-                    releaseExoAudioPlayer();
-                    MediaModel record = new MediaModel(MediaModel.MediaType.AUDIO);
-                    record.setId(UUID.randomUUID().toString());
-                    record.setUrl(filePath);
-                    record.setName(fileName);
-                    initMedia(filePath);
-                    addPostAudioRecord(record);
-                    initializeAudioPlayer();
-                }
+                new RecordingAudioDialog.OnRecordingAudioListener() {
+                    @Override
+                    public void onRecordingAudioClick(String filePath, String fileName) {
+                        releaseExoAudioPlayer();
+                        MediaModel record = new MediaModel(MediaModel.MediaType.AUDIO);
+                        record.setId(UUID.randomUUID().toString());
+                        record.setUrl(filePath);
+                        record.setName(fileName);
```

Comment:
ERROR: 'method def' child have incorrect indentation level 24, expected level should be one of the following: 16, 20.